deploy/examples/keycloak/keycloak.yaml

```diff
@@ -11,4 +11,6 @@ spec:
   externalAccess:
     enabled: True
   podDisruptionBudget:
-    enabled: True
\ No newline at end of file
+    enabled: True
+  externalDatabase:
+    enabled: True
```

Comment:
I'd add an explicit comment here saying that the user has to provision their own DB.